packages/backend/src/app/pieces/pieces.controller.ts

```diff
@@ -34,12 +29,12 @@ export const piecesController: FastifyPluginAsync = async (app) => {
         }
     );
 
-    app.get("/v2/pieces", async () => {
+    app.get("/v1/pieces", async () => {
```

Comment:
Why not keep it /v1/ no one is using it?

Reply:
It's version one already.